src/ray/core_worker/core_worker.h

```diff
@@ -140,14 +140,17 @@ class TaskCounter {
       float running = 0.0;
       float in_get = 0.0;
       float in_wait = 0.0;
+      float alive = 0.0;
       if (running_in_wait_counter_.Total() > 0) {
         in_wait = 1.0;
       } else if (running_in_get_counter_.Total() > 0) {
         in_get = 1.0;
       } else if (num_tasks_running_ > 0) {
         running = 1.0;
+      } else {
+        alive = 1.0;
       }
-      ray::stats::STATS_actors.Record(-(running + in_get + in_wait),
+      ray::stats::STATS_actors.Record(alive,
                                       {{"State", "ALIVE"},
```

Comment:
Should this actually be "IDLE" ?

Reply:
It should but it's a backward incompatible change I think.